### DISCUSSION TITLE
Measurable habits with yearly frequency

### DISCUSSION THREAD

**Discussion by yukienomiya** [iSoron/uhabits#1913] [Feature Requests]
Since it's that time of the year where people set new year's resolutions, it would be handy to have the option to set a measurable habit with "x Every year" frequency. 

So if my goal for the year is to read 24 books I can set that, without missing the 2 books/month goal if I read less books one month and more books the next one.

**Comment by jamesdeluk** [comment]
It could be more granular too, like every quarter, but I think yearly would suffice for most (currently my goal is to run 4.17 hours per month (actually, 50 this year), which is not ideal, especially as it _will_ vary throughout the year as it's weather-dependent).

**Comment by ZornHadNoChoice** [comment]
Even better is to add the custom frequency settings that already exist for yes/no habits. It allows intervals of up to 999 days.

(Extra feature would be to show the remainder to the target divided by the remaining days. For example, if I have 100 days left to my target of learning 2300 kanji and I already know 300, then it would show 20=2000/100. This number is how many kanji I need to learn on average for the remaining days to achieve my goal. Does this deserve its own FR?)

**Comment by herrbrixter** [comment]
This is useful. I have habits that happen rarely like once every quarter.
